main/src/ca/uwaterloo/flix/api/lsp/Entity.scala

```diff
@@ -108,6 +108,12 @@ object Entity {
     def precision: Precision = Precision.High
   }
 
+  case class Label(e: Name.Label) extends Entity {
```

Comment:
Any reason not to take out Field? I would worry that with it present that could be things that have not been renamed correctly?

It should be easy to bring back later.